arbitrum-docs/stylus/stylus-quickstart.md

```diff
@@ -142,7 +142,7 @@ Location:
 
 ```
 
-To read more about what counts as valid vs. invalid user WASM programs, see [VALID_WASM](https://github.com/OffchainLabs/cargo-stylus/blob/main/VALID_WASM.md). If your program succeeds, you'll see the following message:
+To read more about what counts as valid vs. invalid user WASM programs, see [VALID_WASM](https://github.com/OffchainLabs/cargo-stylus/blob/main/check/VALID_WASM.md). If your program succeeds, you'll see the following message:
```

Comment:
This link doesn't seem to be valid. Can you specify what you were pointing to, @fasteater?

Reply:
This link has been updated since.